### ISSUE TITLE
Accessibility problems

### ISSUE THREAD

**Issue by pakjiddat** [exo-addons/task#7]
Hi,

I am running Exo Platform v5 from a Docker container. I installed the task addon using the instruction given on: https://docs.exoplatform.org/PLF50/PLFAdminGuide.AddonsManagement.InstallingUninstallingInDocker.html. I replaced the version number 1.3.x-snapshot with 1.2.4. The addon was installed successfully but the task menu does not show under the Company sidebar. Also when I try to create a new task by clicking on the '+' sign at the top, it allows me to create the task and shows a message saying task was successfully created. It shows a link to the newly created task. But when I click on the task link it does not do anything. Please see the attached screenshots. Thanks

![screenshot from 2017-11-01 17-32-34](https://user-images.githubusercontent.com/20923812/32275031-85c2a480-bf2b-11e7-8dbc-583f519be2ef.png)
![screenshot from 2017-11-01 17-32-51](https://user-images.githubusercontent.com/20923812/32275032-85eccc1a-bf2b-11e7-9779-8b706f58e371.png)
![screenshot from 2017-11-01 17-38-33](https://user-images.githubusercontent.com/20923812/32275033-861b92ac-bf2b-11e7-95bf-486eb2cadacf.png)

**Comment by thomasdelhomenie** [comment]
Right. This issue is already reported here : https://jira.exoplatform.org/browse/TA-573
To solve your problem, if it is a test environment, you can clean the database to start with a fresh installation, it will create the Task menu item.

**Comment by pakjiddat** [comment]
We reinstalled the Exo Platform but are having the same problem. First I installed the latest development version of exo-task and then the latest unstable version. But in both cases it does not show the task navigation menu on the left sidebar.

**Comment by ahamdi** [comment]
@nadirlc You can simply add a new page to the intranet where you put the Tasks application
